engines/pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/jni/JniUtils.java

```diff
@@ -1679,6 +1679,9 @@ public static void deleteModule(long pointer) {
     }
 
     public static void setGraphExecutorOptimize(boolean enabled) {
+        if (enabled) {
```

Comment:
Add log here: https://github.com/deepjavalibrary/djl/blob/master/engines/pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/engine/PtEngine.java#L67

log something like: PyTorch graph executor optimizer is enabled, this may impact your inference latency and throughput. See: https://docs.djl.ai/docs/development/inference_performance_optimization.html#graph-executor-optimization

Reply:
Changed.